test/framework/src/main/java/org/opensearch/test/OpenSearchTestCase.java

```diff
@@ -1768,7 +1768,7 @@ public static String getPortRange() {
         return getBasePort() + "-" + (getBasePort() + 99); // upper bound is inclusive
     }
 
-    protected static int getBasePort() {
+    protected static int generateBasePort(int start) {
```

Comment:
I think you could keep the same method name;

```suggestion
    protected static int getBasePort(int start) {
```